projects/ngrx.io/src/app/app.component.spec.ts

```diff
@@ -618,3 +613,9 @@ class MockAioFooterComponent {
   @Input() nodes;
   @Input() versionInfo;
 }
+
+@Component({
+  selector: 'aio-search-box',
+  template: ''
+})
+class MockAioSearchBoxComponent {}
```

Comment:
This can be removed

Reply:
Done

Reply:
Actually it can't, otherwise, I get this error:

```
'aio-search-box' is not a known element:
	1. If 'aio-search-box' is an Angular component, then verify that it is part of this module.
	2. If 'aio-search-box' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message. ("onSearch)="doSearch($event)" (onFocus)="doSearch($event)"></aio-search-box>
	    </aio-top-menu>
	    [ERROR ->]<aio-search-box class="search-container" *ngIf="!isSideBySide" #searchBox (onSearch)="doSearch($event"): ng:///DynamicTestModule/AppComponent.html@31:4
```